matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/disruptor/UserDisruptorPipeliner.java

```diff
@@ -0,0 +1,42 @@
+package com.matchbook.sdk.core.disruptor;
+
+import com.lmax.disruptor.dsl.Disruptor;
+import com.matchbook.sdk.core.disruptor.handlers.UserPublisherHandler;
+import com.matchbook.sdk.core.disruptor.handlers.UserTransformerHandler;
+import com.matchbook.sdk.core.disruptor.messages.UserMessage;
+
+public class UserDisruptorPipeliner implements AutoCloseable {
+
+    private Disruptor<UserMessage> disruptor;
+    private UserTransformerHandler<UserMessage> userTransformerHandler;
+    private UserPublisherHandler<UserMessage> userPublisherHandler;
+
+    public UserDisruptorPipeliner(Disruptor<UserMessage> disruptor) {
+        this.disruptor = disruptor;
+
+        this.userTransformerHandler = new UserTransformerHandler<>();
+        this.userPublisherHandler = new UserPublisherHandler<>();
+
+        init();
+    }
+
+    private void init() {
```

Comment:
Maybe `buildDisruptor()` could be a better name.

Reply:
i will change it to `initDisruptor`